### PR TITLE
Tones down le solid chems

### PR DESCRIPTION
**Pull request by KayeArray**
:cl: banthebantz
balance: Solid chem and chempile grenade duping toned down significantly, vapour duping slightly buffed
refactor: Vapour runtime caused by reagent ref being GC'd fixed. General refactor to state change multipliers.
/:cl:

Please take note of the now commented out and mirrored reagent splash proc. It was snowflakey oldcode so I had to change it significantly for this to work.

### CODE REVIEW

**Comment by PopNotes** [comment]
it was a good run

thanks for still keeping it to some degree tho

**Comment by GuyonBroadway** [comment]
THis does mean you need to invest some time for a return which I like.

File: code/modules/reagents/chem_splash.dm [hunk 1]
```diff
@@ -4,7 +4,7 @@
 // Threatscale is a multiplier for the 'threat' of the grenade. If you're increasing the affected range drastically, you might want to improve this.
 // Extra heat affects the temperature of the mixture, and may cause it to react in different ways.
 
-
+/*mirrored in hippie folder to fix snowflake splash code bullshit duplication fuck
 /proc/chem_splash(turf/epicenter, affected_range = 3, list/datum/reagents/reactants = list(), extra_heat = 0, threatscale = 1, adminlog = 1)
```
Comment (ghost) [inline code/modules/reagents/chem_splash.dm]: Suggestion: Can we, going forward annotate all hippie changes in /tg/ code with something like this?

```
// HIPPIESTART: Mirrored this function in <file> for <reason>
bunch of shitcode here
// HIPPIESTOP
```

@HippieStation/coders @HippieStation/maintainers what do you guys think?

File: hippiestation/code/modules/reagents/chem_splash.dm [hunk 2]
```diff
@@ -0,0 +1,70 @@
+/proc/chem_splash(turf/epicenter, affected_range = 3, list/datum/reagents/reactants = list(), extra_heat = 0, threatscale = 1, adminlog = 1)
```
Comment (ghost) [inline hippiestation/code/modules/reagents/chem_splash.dm:7]: See my previous comment, any chance you can document the changes done in this function?

**Review by ghost — DISMISSED** [review]
Code looks OK, if you can please address the above comments so that we can actually maintain this.

**Comment by KayeArray** [comment]
Good idea

File: hippiestation/code/modules/reagents/chem_splash.dm (hunk 2 above)
Comment (ghost) [reply hippiestation/code/modules/reagents/chem_splash.dm:7]: @banthebantz

**Review by ghost — DISMISSED** [review]
Code OK

File: code/modules/reagents/chem_splash.dm [hunk 3]
```diff
@@ -4,7 +4,8 @@
 // Threatscale is a multiplier for the 'threat' of the grenade. If you're increasing the affected range drastically, you might want to improve this.
 // Extra heat affects the temperature of the mixture, and may cause it to react in different ways.
 
-
+// HIPPIESTART: Mirrored this function in <hippiestation\code\modules\reagents\chem_splash.dm> for <reducing the duplication of solid and turchchems by refactoring chem splash code to use a particle_effect holder atom akin to smoke>
```
Comment (ghost) [inline code/modules/reagents/chem_splash.dm]: You've made changes to /tg/ files, please mark these changes.

**All modifications to non-hippie files should be marked.**

- Multi line changes start with `// hippie start` and end with `// hippie end`
- You can put a messages with a change if it isn't obvious, like this: `// hippie start - reason`
  - Should generally be about the reason the change was made, what it was before, or what the change is
  - Multi-line messages should start with `// hippie start` and use `/* Multi line message here */` for the message itself
- Single line changes should have `// hippie` or `// hippie - reason`

If you need to mirror a file, or function into a hippie-specific file, please leave behind a comment stating where it went.

```
// hippie start - Mirrored this function in <file> for <reason>
bunch of shitcode here
// hippie stop
```

Once you mirror a file, please follow the above for marking your changes, this way we know what needs to be updated when a file has been mirrored.

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by dreamcrab** [comment]
cool +7685

File: code/modules/reagents/chem_splash.dm (hunk 1 above)
Comment (ghost) [reply code/modules/reagents/chem_splash.dm]: i think DON'T PING ME

**Review by ghost — APPROVED** [review]
Code OK

**Comment by ghost** [comment]
ok

**Comment by ghost** [comment]
GO FUCKO DO IT +1

**Comment by GuyonBroadway** [comment]
Thats a good review, a +3 and nearly a weeks worth of waiting. Merger a gogo

**Pull request merged**